logstash-core/src/main/java/org/logstash/execution/AbstractPipelineExt.java

```diff
@@ -345,35 +343,24 @@ public final PipelineReporterExt reporter() {
     @JRubyMethod(name = "collect_dlq_stats")
     public final IRubyObject collectDlqStats(final ThreadContext context) {
         if (dlqEnabled(context).isTrue()) {
-            getDlqMetric(context).gauge(
-                    context,
-                    QUEUE_SIZE_IN_BYTES_KEY,
-                    dlqWriter(context).callMethod(context, "get_current_queue_size")
-            );
-            getDlqMetric(context).gauge(
-                    context,
-                    STORAGE_POLICY_KEY,
-                    dlqWriter(context).callMethod(context, "get_storage_policy")
-            );
-            getDlqMetric(context).gauge(
-                    context,
-                    MAX_QUEUE_SIZE_IN_BYTES_KEY,
-                    getSetting(context, "dead_letter_queue.max_bytes").convertToInteger());
-            getDlqMetric(context).gauge(
-                    context,
-                    DROPPED_EVENTS_KEY,
-                    dlqWriter(context).callMethod(context, "get_dropped_events")
-            );
-            getDlqMetric(context).gauge(
-                    context,
-                    LAST_ERROR_KEY,
-                    dlqWriter(context).callMethod(context, "get_last_error")
-            );
-            getDlqMetric(context).gauge(
-                    context,
-                    EXPIRED_EVENTS_KEY,
-                    dlqWriter(context).callMethod(context, "get_expired_events")
-            );
+            getDlqMetric(context).gauge(context,
```

Comment:
I have pending checkstyle apply [draft PR](https://github.com/elastic/logstash/pull/14251) would be nice to push back :( We all have different rules on local.